source/base/parameter_handler.cc

```diff
@@ -1219,18 +1219,28 @@ ParameterHandler::set(const std::string &entry_string, const bool new_value)
 
 std::ostream &
 ParameterHandler::print_parameters(std::ostream &    out,
-                                   const OutputStyle style,
-                                   const bool        sort_alphabetical) const
+                                   const OutputStyle style) const
 {
   AssertThrow(out, ExcIO());
 
+  Assert(
+    (((style & XML) > 0) + ((style & JSON) > 0) + ((style & PRM) > 0) +
+     ((style & Description) > 0) + ((style & LaTeX) > 0)),
```

Comment:
Isn't a `==1` missing?
```suggestion
    (((style & XML) > 0) + ((style & JSON) > 0) + ((style & PRM) > 0) +
     ((style & Description) > 0) + ((style & LaTeX) > 0)) == 1,
```

Reply:
Funny question that I'm not sure about: Say, `OutputStyle` uses `short unsigned int` as base type. The `style & XML` is promoted to `int`. Is the result guaranteed to be `>0` is one bit is set, or could this result in the sign bit being set and the result is negative? I think what you *want* to test is that at least one bit is set, so maybe using `!= 0` is the better check over `>0`?